docs/src/pages/demos/snackbars/IntegrationNotistack.tsx

```diff
@@ -27,6 +23,10 @@ class App extends React.Component<withSnackbarProps> {
   }
 }
 
+(App as React.ComponentClass<withSnackbarProps>).propTypes = {
```

Comment:
This is quite ugly but it's the pattern we use throughout the codebase. Allows separation of interface and implementation and doesn't block readability (usually what we're interested in first is `render`).